### DISCUSSION TITLE
Beautifying multiple files (HTML)

### DISCUSSION THREAD

**Discussion by fermiongithub** [brackets-cont/brackets#276] [Q&A]
Hey : )

I like to beautify all files (HTML) of a folder and its subfolders (not only one single file), the files of a complete site.

Is that possible with any available extension?

https://github.com/brackets-beautify/brackets-beautify
https://github.com/hirse/brackets-prettier
https://github.com/dingdong-io/brackets.beautify.io
...

It would be great, when normalization of whitepaces is part of the beautifying.

I use the latest version of Brackets for macOS: Release 2.2 Build 2.2.0-17942

Thanks for your help : )

**Comment by charlypa** [comment]
Currently, brackets extensions focus on single file beautification. However, your suggestion for entire site beautification is insightful. If widely requested, we could add it to our future plans.

Thanks for your input!

**Comment by fermiongithub** [comment]
@charlypa
Thanks for your answer.
You write: "brackets extensions focus on single file beautification".
Does this also apply to other extensions, where it is not about beautification? Is it not possible so far to let an extension work on more than one file?
